frameworks/elastic/src/main/dist/elasticsearch.yml

```diff
@@ -54,6 +54,12 @@ xpack.security.http.ssl.enabled: true
 xpack.security.http.ssl.client_authentication: none
 {{/SECURITY_TRANSPORT_ENCRYPTION_ENABLED}}
 
+# FIXME: Temporarily disabling X-Pack ML module because it looks like our task containers are missing a few system
```

Comment:
Is this tracked in a JIRA ticket?

Reply:
I thought I had a ticket but didn't. Just created one: https://jira.mesosphere.com/browse/ELASTIC-169